integration_tests/test_redis_myco_matcher.py

```diff
@@ -37,3 +37,15 @@ def on_matched_recommendations_response(self, data):
 
     def on_event_or_response(self, data):
         self.called_events.add("event_or_response")
+
+    def on_area_map(self, data):
+        area_list = ["Grid", "House 1", "H1 General Load1", "H1 General Load2",
+                     "H1 Storage1", "H1 Storage2", "H1 PV1", "H1 PV2",
+                     "House 2", "H2 General Load1", "H2 Storage1", "H2 PV", "Cell Tower",
+                     "Market Maker"]
+        if data == {}:
```

Comment:
Nit, these can be grouped in one check
```suggestion
        if not (data.get("area_map") and list(data["area_map"].values()) == areas_list):
            self.errors += 1
```